less/about/newtab.less

```diff
@@ -45,10 +44,6 @@ ul {
   }
 }
 
-#appContainer > div {
-  height: auto; // overrides default #appContainer > div: 100%
-}
-
```

Comment:
This has to be removed; It looks like starting with Chromium 56, changes were made to `background-image` which causes the image to render itself at 100% DPI (rather than scaling to the size of the window)